src/ktopt_planning_context.cpp

```diff
@@ -137,19 +121,38 @@ void KTOptPlanningContext::solve(planning_interface::MotionPlanResponse& res)
     return;
   }
 
+  RCLCPP_INFO(getLogger(), "Setting initial guess ...");
+  // set the initial guess
+  trajopt.SetInitialGuess(trajopt.ReconstructTrajectory(result));
+  // const auto& collision_constraint = MinimumDistanceLowerBoundConstraint(&plant_, 0.01, &plant_context_);
+
+  // add collision constraints
+  for (double s = 0.0; s <=25.0; s++)
+  {
+    trajopt.AddPathPositionConstraint(
+      std::make_shared<MinimumDistanceLowerBoundConstraint>(
+        &plant_,
+        0.01,
+        &plant_context_),
+      s/25.0);
+  }
```

Comment:
turn this 0.01 and 25.0 into parameters -- or at least for now constants defined in the anonymous namespace up top.